baseapp/abci.go

```diff
@@ -795,35 +819,6 @@ func handleQueryStore(app *BaseApp, path []string, req abci.RequestQuery) abci.R
 	return resp
 }
 
-func handleQueryP2P(app *BaseApp, path []string) abci.ResponseQuery {
```

Comment:
we need to keep this since it's used by `Query`

Reply:
This was moved to `baseapp/p2p.go`. Is that still an issue?